item note added (#3433)
jenkins-infra/jenkins.io: content/doc/book/installing/_setup-wizard.adoc

image:tutorials/setup-jenkins-02-copying-initial-admin-password.png[alt="Copying initial admin password",width=80%] +
  *Note:* 
* The command: `sudo cat /var/lib/jenkins/secrets/initialAdminPassword` will print the password at console. +

> * If you running Jenkins by a docker container you can use `sudo docker exec jenkins-blueocean cat /var/jenkins_home/secrets/initialAdminPassword` to print the password at console without get inside it. The `jenkins-blueocean` it is the link:http://localhost:4242/doc/book/installing/#downloading-and-running-jenkins-in-docker[container-name user in our docs sample], just remember to replace it to your own if you pick a different one. +

Reviewer: This `If you running Jenkins by a docker container` should be `If you running Jenkins in a docker container`

Author: tks

> * If you running Jenkins by a docker container you can use `sudo docker exec jenkins-blueocean cat /var/jenkins_home/secrets/initialAdminPassword` to print the password at console without get inside it. The `jenkins-blueocean` it is the link:http://localhost:4242/doc/book/installing/#downloading-and-running-jenkins-in-docker[container-name user in our docs sample], just remember to replace it to your own if you pick a different one. +

Reviewer: This `to print the password at console without get inside it.` should be `to print the password in the console without having to exec into the container.`

> * If you running Jenkins by a docker container you can use `sudo docker exec jenkins-blueocean cat /var/jenkins_home/secrets/initialAdminPassword` to print the password at console without get inside it. The `jenkins-blueocean` it is the link:http://localhost:4242/doc/book/installing/#downloading-and-running-jenkins-in-docker[container-name user in our docs sample], just remember to replace it to your own if you pick a different one. +

Reviewer: This `just remember to replace it to your own if you pick a different one` should be `just remember to replace it with your own if you pick a different one`

> * If you running Jenkins in a docker container you can use `sudo docker exec jenkins-blueocean cat /var/jenkins_home/secrets/initialAdminPassword` to print the password in the console without having to exec into the container. The `jenkins-blueocean` it is the link:http://localhost:4242/doc/book/installing/#downloading-and-running-jenkins-in-docker[container-name user in our docs sample], just remember to replace it with your own if you pick a different one. +

Reviewer 2: ```suggestion
* If you are running Jenkins in Docker using the official `jenkins/jenkins` image you can use `sudo docker exec ${CONTAINER_ID or CONTAINER_NAME} cat /var/jenkins_home/secrets/initialAdminPassword` to print the password in the console without having to exec into the container.
```

Author: Did it. Thank you.
